bazel/repository_locations.bzl

```diff
@@ -237,13 +237,15 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "LuaJIT",
         project_desc = "Just-In-Time compiler for Lua",
         project_url = "https://luajit.org",
-        version = "2.1.0-beta3",
-        sha256 = "409f7fe570d3c16558e594421c47bdd130238323c9d6fd6c83dedd2aaeb082a8",
+        # The last release version, 2.1.0-beta3 has a number of CVEs filed
+        # against it. These may not impact correct non-malicious Lua code, but for prudence we bump.
+        version = "e9af1abec542e6f9851ff2368e7f196b6382a44c",
+        sha256 = "186cd55c0e321aa2ec3c0f9220e31cc17479ec6222db515ec8b902f9275a9616",
         strip_prefix = "LuaJIT-{version}",
-        urls = ["https://github.com/LuaJIT/LuaJIT/archive/v{version}.tar.gz"],
+        urls = ["https://github.com/LuaJIT/LuaJIT/archive/{version}.tar.gz"],
+        last_updated = "2020-10-09",
```

Comment:
This commit is 2020-09-29 and `last_updated` should reflect that. Consider bumping to latest commit `1d8b747c161db457e032a023ebbff511f5de5ec2` `2020-10-12`.